ApplicationLibCode/ProjectDataModel/StimPlanModel/RimStimPlanModelPressureCalculator.h

```diff
@@ -58,11 +60,29 @@ class RimStimPlanModelPressureCalculator : public RimStimPlanModelWellLogCalcula
                                        std::vector<double>&      measuredDepthValues,
                                        std::vector<double>&      tvDepthValues ) const;
 
-    bool interpolateInitialPressureByEquilibrationRegion( const RimStimPlanModel*    stimPlanModel,
+    bool interpolateInitialPressureByEquilibrationRegion( RiaDefines::CurveProperty  curveProperty,
+                                                          const RimStimPlanModel*    stimPlanModel,
                                                           int                        timeStep,
                                                           const std::vector<double>& measuredDepthValues,
                                                           const std::vector<double>& tvDepthValues,
                                                           std::vector<double>&       values ) const;
+    bool interpolatePressureDifferenceByEquilibrationRegion( RiaDefines::CurveProperty  curveProperty,
+                                                             const RimStimPlanModel*    stimPlanModel,
+                                                             int                        timeStep,
+                                                             const std::vector<double>& measuredDepthValues,
+                                                             const std::vector<double>& tvDepthValues,
+                                                             const std::vector<double>& initialPressureValues,
+                                                             std::vector<double>&       values ) const;
+    bool handleFaciesWithInitialPressure( const RimStimPlanModel*    stimPlanModel,
+                                          int                        timeStep,
+                                          const std::vector<double>& faciesValues,
+                                          std::vector<double>&       values ) const;
+
+    static std::tuple<std::vector<double>, std::vector<double>, std::vector<double>>
+        interpolateMissingValues( const std::vector<double>& staticTvDepthValues,
+                                  const std::vector<double>& staticMeasuredDepthValues,
+                                  const std::vector<double>& measuredDepthValues,
+                                  const std::vector<double>& values );
```

Comment:
Please find better name for variable values

Reply:
Fixed.